packages/react-core/src/components/DatePicker/DatePicker.tsx

```diff
@@ -272,7 +272,7 @@ const DatePickerBase = (
         </div>
       </Popover>
       {helperText && <div className={styles.datePickerHelperText}>{helperText}</div>}
-      {errorText.trim() && <div className={css(styles.datePickerHelperText, styles.modifiers.error)}>{errorText}</div>}
+      {errorText.trim() && <div className={css(styles.datePickerHelperText)}>{errorText}</div>}
```

Comment:
Won't this cause error to be unstyled (or styled the same way as helper)?

Reply:
![image](https://user-images.githubusercontent.com/5322142/226022382-b9a2f69d-2dca-4b06-b4b9-07e2f37cd4bf.png)

looks like it is also causing cypress to fail as it tries to find `div.pf-m-error` 
https://github.com/patternfly/patternfly-react/blob/v5/packages/react-integration/cypress/integration/datepicker.spec.ts#L17

Reply:
There is a follow up issue for the helper text work

Reply:
I disabled the test for now.  It should be reenabled with the datePicker helper text work (I left a note in the issue).  This PR is just to bump the core version and get it to build so we have the styles to address the feature work once this goes in